### DISCUSSION TITLE
Configure Zitadel for Mealie

### DISCUSSION THREAD

**Discussion by Aurel004** [zitadel/zitadel#7905] [Show and tell]
Hi,

I made a tutorial on how to setup Zitadel for Mealie, you can have a look here: https://github.com/mealie-recipes/mealie/discussions/3557

**Comment by fforootd** [comment]
Wow nice, thanks for sharing 🤩

**Reply by Aurel004** [reply]
Doesn't Zitadel store all the tutorials for specific applications (like this one) ?
A bit like Authelia and Authentik do

**Reply by fforootd** [reply]
We have a section in our docs for this https://zitadel.com/docs/guides/integrate/services

Should not be too much effort.

1) Create a new tool here https://github.com/zitadel/zitadel/tree/main/docs/docs/guides/integrate/tools
2) Create a link here https://github.com/zitadel/zitadel/blob/1c5ecba42a8ffb5b366db9c5ca28a7170cdd33a1/docs/sidebars.js#L430
